.travis.yml

```diff
@@ -50,9 +50,15 @@ before_install:
     fi
 script:
   - rustc --version
-  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.4" ]; then
+  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.14" ]; then
     make regen_check;
     fi
+  - cargo doc --features "dox,embed-lgpl-docs"
+  - cargo test --features "$FEATURES,embed-lgpl-docs"
+  # catch any sneaked in lgpl docs
+  - cargo build --features "$FEATURES,purge-lgpl-docs" --jobs 1
+  - git diff -R --exit-code
+  - rustc --version
```

Comment:
Hm, this really needed here? Why not "rust -Vv" ?

Reply:
Just version printed at begining: https://travis-ci.org/gtk-rs/pango/jobs/489641332#L595

Reply:
That's how all the `.travis.yml` are. That's why I mentioned before that we need something like the gitlab-ci templates, or otherwise everything will get out of sync or we have to fix the same thing 10 times.